extracter/server.mjs

```diff
@@ -0,0 +1,47 @@
+import compression from 'compression';
+import cors from 'cors';
+import express from 'express';
+
+export class Server {
```

Comment:
There are two `Server` classes, is this class necessary ?

Reply:
it's not, I forgot to clean it up